packages/11ty/content/_assets/javascript/application/canvas-panel.js

```diff
@@ -1,51 +1,107 @@
-window.onload = function () {
-  const canvasPanels = document.getElementsByTagName('canvas-panel')
+import scrollToHash from "./scroll-to-hash"
+
+/**
+ * Handle annotation/choice selection with canvasPanel API
+ * {@link https://iiif-canvas-panel.netlify.app/docs/examples/handling-choice}
+ */
+const handleSelect = (element) => {
+  const figure = element.closest('.q-figure')
+  if (!figure) return
+  const canvasPanel = figure.querySelector('canvas-panel')
+  const annotationType = element.getAttribute('data-annotation-type')
+  const id = element.getAttribute('id')
+  const inputType = element.getAttribute('type')
+
+  const toggleChoice = ({ checked }) => {
+    canvasPanel.makeChoice(id, {
+      deselect: !checked,
+      deselectOthers: inputType === 'radio'
+    })
+    if (inputType !== 'checkbox') return
```

Comment:
Whenever possible, the guard clause be the first statement, for example:
```suggestion
    if (inputType !== 'checkbox') return
    
    canvasPanel.makeChoice(id, {
      deselect: !checked,
      deselectOthers: inputType === 'radio'
    })
```

Reply:
that guard clause is only guarding the things below it, maybe there is another way to write this though